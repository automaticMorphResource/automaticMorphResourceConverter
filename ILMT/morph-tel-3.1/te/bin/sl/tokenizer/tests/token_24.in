అడిలైడ్ నగరానికి దాదాపు 100 కిలోమీటర్ల దూరంలో ఉండే "కేప్ జర్విస్" అనే రేవు నుండి సముద్రంలో దాదాపు 15 కిలోమీటర్ల మేర ఇది ఉంటుంది. 
