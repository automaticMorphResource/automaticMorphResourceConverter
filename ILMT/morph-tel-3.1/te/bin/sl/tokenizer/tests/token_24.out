<Sentence id="1">
1	అడిలైడ్	unk
2	నగరానికి	unk
3	దాదాపు	unk
4	100	unk
5	కిలోమీటర్ల	unk
6	దూరంలో	unk
7	ఉండే	unk
8	"	unk
9	కేప్	unk
10	జర్విస్	unk
11	"	unk
12	అనే	unk
13	రేవు	unk
14	నుండి	unk
15	సముద్రంలో	unk
16	దాదాపు	unk
17	15	unk
18	కిలోమీటర్ల	unk
19	మేర	unk
20	ఇది	unk
21	ఉంటుంది	unk
22	.	unk
</Sentence>

